lib/services/internalEngine/index.js

```diff
@@ -293,17 +316,17 @@ class InternalEngine {
     return this.exists(type, id)
       .then(exists => {
         const request = {
-          index: this.index,
           type,
           id,
+          index: this.index,
           body: content
         };
 
         if (exists) {
           return this.client.index(request);
         }
 
-        return Promise.reject(new NotFoundError('Document with id ' + id + ' not found.'));
+        return Bluebird.reject(new NotFoundError(`Document with id "${id}" not found.`));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 89 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)